### PR TITLE
#1459 Update when INVEST NOW button is shown

### PR DESCRIPTION
**Pull request by desfero**

### CODE REVIEW

File: app/components/eto/overview/EtoOverviewStatus/InvestmentWidget/InvestmentWidget.tsx [hunk 1]
```diff
@@ -56,36 +64,47 @@ const InvestmentWidgetLayout: React.SFC<TInvestWidgetProps> = ({
...
-              >
-                <FormattedMessage id="shared-component.eto-overview.invest-now" />
-              </Button>
-            )
-          }
-        </EtoWidgetContext.Consumer>
-      </div>
+      <EtoWidgetContext.Consumer>
+        {previewCode =>
+          (isInvestor || previewCode) && (
```
Comment (desfero) [inline app/components/eto/overview/EtoOverviewStatus/InvestmentWidget/InvestmentWidget.tsx]: Here is a fix for a state when you are logged in as issuer. Previously `Invest now` button was also shown for issuer

File: app/modules/notifications/selectors.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ export const selectIsActionRequiredSettings = (state: IAppState): boolean => {
     !selectIsUserEmailVerified(state.auth) ||
     (!selectBackupCodesVerified(state.auth) &&
       selectWalletType(state.web3) === EWalletType.LIGHT) ||
-    indexOf(["Outsourced", "Pending", "Accepted"], selectKycRequestStatus(state.kyc)) === -1
+    !includes(["Outsourced", "Pending", "Accepted"], selectKycRequestStatus(state.kyc))
```
Comment (desfero) [inline app/modules/notifications/selectors.ts]: `includes` is more developer friendly comparing to `indexOf`

File: yarn.lock [hunk 3]
```diff
@@ -9626,7 +9626,7 @@ ledger-wallet-provider@^2.0.1-beta.4:
     strip-hex-prefix "^1.0.0"
     web3-provider-engine "^13.3.3"
 
-"ledgerco@github:Neufund/ledgerjs#tomek/update_hid":
+ledgerco@Neufund/ledgerjs#tomek/update_hid:
```
Comment (desfero) [inline yarn.lock]: Maybe @banciur use older yarn version? 🤔

File: app/components/eto/overview/EtoOverviewStatus/InvestmentWidget/InvestmentWidget.tsx (hunk 1 above)
Comment (AlexeyTuboltsev) [reply app/components/eto/overview/EtoOverviewStatus/InvestmentWidget/InvestmentWidget.tsx]: 👍

File: app/components/eto/EtoDashboard.tsx [hunk 4]
```diff
@@ -299,7 +299,6 @@ class EtoDashboardComponent extends React.Component<IProps> {
 export const EtoDashboard = compose<React.SFC>(
   appConnect<IStateProps, IDispatchProps>({
     stateToProps: s => ({
-      kycStatus: selectKycRequestStatus(s.kyc),
```
[the remark this answers is not in the document]
Comment (trival) [reply app/components/eto/EtoDashboard.tsx]: 👍

**Pull request merged**